pkgs/development/python-modules/frameioclient/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, urllib3
+, importlib-metadata
+}:
+
+buildPythonPackage rec {
+  pname = "frameioclient";
+  version = "0.7.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "3ec2dd0be305a6249f365b23210ec8f554288752899bceae82c133264c720092";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    urllib3
+    importlib-metadata
+  ];
```

according to https://github.com/Frameio/python-frameio-client/blob/9770f660a53541cd0790ffed4863b1bbb19c7a0c/setup.py#L29
```suggestion
  propagatedBuildInputs = [
    requests
    urllib3 ] ++ lib.optionals (pythonOlder "3.8") [
    importlib-metadata
  ];
```